### PR TITLE
doc: Add a missing closing parenthesis in the bitcoin-wallet's help

### PR DESCRIPTION
**Pull request by darosior**

### CODE REVIEW

File: src/bitcoin-wallet.cpp [hunk 1]
```diff
@@ -27,7 +27,7 @@ static void SetupWalletToolArgs()
     gArgs.AddArg("-datadir=<dir>", "Specify data directory", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-wallet=<wallet-name>", "Specify wallet name", ArgsManager::ALLOW_ANY | ArgsManager::NETWORK_ONLY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-debug=<category>", "Output debugging information (default: 0).", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
-    gArgs.AddArg("-printtoconsole", "Send trace/debug info to console (default: 1 when no -debug is true, 0 otherwise.", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
+    gArgs.AddArg("-printtoconsole", "Send trace/debug info to console (default: 1 when no -debug is true, 0 otherwise.)", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
```
Comment (kristapsk) [inline src/bitcoin-wallet.cpp]: Shouldn't full stop go after closing parenthesis not before?

**Review by kristapsk — CHANGES_REQUESTED** [review]

File: src/bitcoin-wallet.cpp (hunk 1 above)
Comment (darosior) [reply src/bitcoin-wallet.cpp]: Thanks

**Review by kristapsk — APPROVED** [review]

**Review by kristapsk — APPROVED** [review]
utACK b6233a4985519fea2ca35ae6d45877bb2df697f9

**Comment by emilengler** [comment]
Concept ACK

**Comment by promag** [comment]
ACK

**Review by fanquake — APPROVED** [review]
ACK b6233a4985519fea2ca35ae6d45877bb2df697f9

**Pull request merged**